share_your_route_front/lib/map.dart

```diff
@@ -0,0 +1,116 @@
+// ignore_for_file: constant_identifier_names
+
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:latlong2/latlong.dart';
+import 'package:share_your_route_front/classes/tourist_route.dart';
+
+const MAPBOX_ACCESS_TOKEN =
+    'sk.eyJ1IjoiZ2phcmV2YWwiLCJhIjoiY2x4MThna3hzMDhqZDJxcTdjaXFxc29vaSJ9.xP7hhukr96mBVQ6aakeAFg';
```

Comment:
Utilicen variables de entorno para esto

https://medium.com/@eng.abdalrhman33/use-environment-variables-for-flutter-apps-fd9d02289076